presto-docs/src/main/sphinx/release/release-0.153.rst

```diff
@@ -6,32 +6,154 @@ General Changes
 ---------------
 
 * Fix incorrect results for grouping sets when ``task.concurrency`` is greater than one.
```

Comment:
I'm not sure we should update everything. There are other bugs like `try_cast` that would require adding warnings to dozens of releases. I think it's understood that each release can fix correctness bugs, and if you continue to run an old release, you should see what is fixed in newer releases.